### DISCUSSION TITLE
User Identifiants

### DISCUSSION THREAD

**Discussion by lucile98-ode** [lucile98-ode/GPI-L2BHA-FENOUIL#13] [Ideas]
User Base de données IDENTIFIANT:

1 OBER DYLAN Entreprise
2 MOUSE MINNIE Gestionnaire
3 DUCK DONALD Responsable
4 DUCK DAISY Assistant
5 MOUSE MICKEY Directeur

**Comment by lucile98-ode** [comment]
Les acteurs de l'application:
- Assistant de saisie
- Gestionnaire administratif
- responsable de routage
- directeur de la stratégie
- Membre du département de Prospection
- Entreprise (saisie de commande)

**Reply by chrabou** [reply]
Toute ses personnes seront intégrés sur l'application

**Reply by amrx12** [reply]
Les acteurs sont bien présent, ils accèdent à leur menu via un espace de connexion qui leur est dédié directement.

**Comment by lucile98-ode** [comment]
La Base de données Finale regroupera TOUTE la Base de données mais avec plusieurs tables différentes, ainsi il faudra appeler par commande SQL les différentes tables (comme durant le cours de Base de données)

@lucile98-ode @chrabou @amrx12 @HaceneM
